modules/salt/pages/large-scale.adoc

```diff
@@ -174,3 +174,15 @@ If significant non-transient swap usage is detected, you can increase the availa
 
 You can also consider tuning the Server to consume less memory.
 For more information on tuning, see xref:salt:large-scale-tuning.adoc[].
+
+=== AES Key rotation
+
+Communication from the Salt Master to all Salt minions is encrypted with a single AES key. This key is rotated whenever:
+ - the salt-master process is restarted, or
+ - any Salt minion key is deleted (eg. a client is deleted from {productname})
+
+Upon rotation, all minions will have to re-authenticate to the Master next time they receive a message, and if the number of minions is high (several thousands) then this is expected to cause noticeable CPU load on the {productname} Server.
```

Comment:
```suggestion
After the AES key has been rotated, all clients must re-authenticate to the master.
By default, this happens next time a client receives a message.
If you have a large number of clients (several thousands), this can cause a high CPU load on the {productname} Server.
```